plugins/html/icon_html/util/helpers.py

```diff
@@ -17,8 +21,44 @@ def read_file_content(filename: str) -> str:
         return encode_to_base64(file_.read())
 
 
+def delete_file(file_name):
+    try:
+        os.remove(file_name)
+    except FileNotFoundError:
+        pass
+    except Exception as e:
```

Comment:
Iirc best practice calls for catching the most specific exception you can rather than just `Exception` and calling it a day.